src/js/components/FormField/doc.js

```diff
@@ -175,6 +175,42 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: undefined,
   },
+  'formField.error.size': {
+    description: `The size of the error messsage to be displayed.
+     The default size is medium.`,
+    type: 'string',
+    defaultValue: 'medium',
+  },
+  'formField.error.size.xsmall': {
+    description: `The size of the error messsage to be displayed.
+     The default size is medium.`,
```

Comment:
```suggestion
    description: `The size of a 'xsmall' error message.`,
```
The same fix for description should be applied to the rest of the sizes.
Also, there is a type for  'messsage' that should be 'message' instead across the other sizes..

Reply:
All done, Thank You for all your help and suggestions.